pkg/rancher-desktop/integrations/windowsIntegrationManager.ts

```diff
@@ -234,7 +234,7 @@ export default class WindowsIntegrationManager implements IntegrationManager {
       reason = 'not enforcing';
     } else if (!this.backendReady) {
       reason = 'backend not ready';
-    } else if (this.settings.kubernetes?.containerEngine !== ContainerEngine.MOBY) {
+    } else if (this.settings.containerEngine?.name !== ContainerEngine.MOBY) {
       reason = `unsupported container engine ${ this.settings.kubernetes?.containerEngine }`;
```

Comment:
`containerEngine` no longer hangs off `kuberenetes`.

Reply:
yes, another one that eluded my diff

Reply:
Caught this before your review.